tests/test_utils.py

```diff
@@ -0,0 +1,42 @@
+import cProfile
```

Comment:
Please remove this file. We don't test any code correctness and profiling can be done without this file.

Reply:
Moved it to `performance_tests/`. Is that ok? I think performance testing is important.
commit 2c7ac1a

Reply:
Works for me. It's clear they're not regular tests and they won't be run by CI or pre-commit hooks.